testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/jms/ClusteredMessagingTestCase.java

```diff
@@ -46,145 +46,118 @@
-        }
-        try (ModelControllerClient client = createClient2()) {
-            JMSOperations jmsOperations = JMSOperationsProvider.getInstance(client);
-            jmsOperations.removeJmsQueue(jmsQueueName);
-            jmsOperations.removeJmsTopic(jmsTopicName);
-        }
+        });
     }
 
     @Test
     public void testClusteredQueue() throws Exception {
-        InitialContext contextFromServer0 = createJNDIContextFromServer0();
-        InitialContext contextFromServer1 = createJNDIContextFromServer1();
+        InitialContext contextFromServer1 = createJNDIContext(NODE_1);
+        InitialContext contextFromServer2 = createJNDIContext(NODE_2);
 
         String text = UUID.randomUUID().toString();
 
-        // WIP test if the problem is that the view is not yet propagated
-        Thread.sleep(GRACE_TIME_TO_MEMBERSHIP_CHANGE);
-
-        // send to the queue on server 0
-        sendMessage(contextFromServer0, jmsQueueLookup, text);
-        // receive it from the queue on server 1
-        receiveMessage(contextFromServer1, jmsQueueLookup, text);
+        // send to the queue on server 1
+        sendMessage(contextFromServer1, jmsQueueLookup, text);
+        // receive it from the queue on server 2
+        receiveMessage(contextFromServer2, jmsQueueLookup, text);
 
         String anotherText = UUID.randomUUID().toString();
-        // send to the queue on server 1
-        sendMessage(contextFromServer1, jmsQueueLookup, anotherText);
-        // receive it from the queue on server 0
-        receiveMessage(contextFromServer0, jmsQueueLookup, anotherText);
+        // send to the queue on server 2
+        sendMessage(contextFromServer2, jmsQueueLookup, anotherText);
+        // receive it from the queue on server 1
+        receiveMessage(contextFromServer1, jmsQueueLookup, anotherText);
     }
 
     @Test
     public void testClusteredTopic() throws Exception {
-
-        InitialContext contextFromServer0 = createJNDIContextFromServer0();
-        InitialContext contextFromServer1 = createJNDIContextFromServer1();
+        InitialContext contextFromServer1 = createJNDIContext(NODE_1);
+        InitialContext contextFromServer2 = createJNDIContext(NODE_2);
 
         try (
-                JMSContext jmsContext0 = createJMSContext(createJNDIContextFromServer0());
-                JMSContext jmsContext1 = createJMSContext(createJNDIContextFromServer1())
+                JMSContext jmsContext1 = createJMSContext(createJNDIContext(NODE_1));
+                JMSContext jmsContext2 = createJMSContext(createJNDIContext(NODE_2))
         ) {
-            JMSConsumer consumer0 = jmsContext0.createConsumer((Destination) contextFromServer0.lookup(jmsTopicLookup));
             JMSConsumer consumer1 = jmsContext1.createConsumer((Destination) contextFromServer1.lookup(jmsTopicLookup));
+            JMSConsumer consumer2 = jmsContext2.createConsumer((Destination) contextFromServer2.lookup(jmsTopicLookup));
```

Comment:
Those consumers being Autoclseable they should be in the 'try-with-resources'

Reply:
Done.